rb/lib/selenium/webdriver/remote.rb

```diff
@@ -31,7 +31,8 @@ module Remote
       autoload :Capabilities, 'selenium/webdriver/remote/capabilities'
 
       module Http
-        autoload :Common,  'selenium/webdriver/remote/http/common'
+        autoload :Common, 'selenium/webdriver/remote/http/common'
+        autoload :Curb, 'selenium/webdriver/remote/http/curb'
```

Comment:
Can you also update the `@example` in the client and remove unnecessary `require/include`?